lib/handlers/cypress_mongo_handler.js

```diff
@@ -0,0 +1,168 @@
+const mongoose = require('mongoose');
+const IndividualResultSchema = require('cqm-models').IndividualResultSchema;
+
+module.exports = class Handler {
```

Comment:
There should be some type of super class "Handler" which specific handlers "extend" from.  I recommend naming this to something more cypress specific.